sixtyfps_runtime/rendering_backends/gl/lib.rs

```diff
@@ -457,7 +457,9 @@ fn rect_with_radius_to_path(rect: Rect, border_radius: f32) -> femtovg::Path {
     // If we're drawing a circle, use directly connected bezier curves instead of
     // ones with intermediate LineTo verbs, as `rounded_rect` creates, to avoid
     // rendering artifacts due to those edges.
-    if width == height && border_radius * 2. == width {
+    if (width - height).abs() < 10.0 * f32::EPSILON
```

Comment:
I’m curious, where does the 10 come from?

Reply:
It seemed small enough...